starter-code/routes/auth.js

```diff
@@ -0,0 +1,17 @@
+const express = require('express')
+const routerA = express.Router()
+const passport = require('passport')
+const LocalStrategy = require('passport-local').Strategy;
+const User = require('../models/user');
+const bcrypt = require('bcrypt');
+
+routerA.get('/login', (req, res, next) => {
+    res.render('auth/login')
+})
+
+routerA.post('/login', passport.authenticate('local', {
+    successRedirect: '/main',
```

Comment:
You don't have a '/main' route set up. Everything works but in the future it would be better if you had a route that would send users to their profile after logging in. Great work though!